package.json

```diff
@@ -63,10 +59,10 @@
     "detect-node": "^2.0.4",
     "err-code": "^3.0.0",
     "libp2p-interfaces": "libp2p/js-interfaces#chore/skip-abort-while-reading-for-webrtc",
```

Comment:
Should be removed before release

Reply:
This is an old thing that needs to be fixed. This comes from the async refactor: https://github.com/libp2p/js-libp2p-webrtc-direct/pull/30

I will create an issue so that we can triage it, but tldr there is a bug in this module when we abort while reading

Reply:
https://github.com/libp2p/js-libp2p-webrtc-direct/issues/117